test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -80,4 +108,32 @@ Describe "Import-Alias" -Tags "CI" {
 	    (ipal $pesteraliasfile)
 	    (pesterecho pestertesting) | Should -BeExactly "pestertesting"
 	}
+
+	It "Should be able to parse ""abc""""def"" into abc""def " {
+		(Import-Alias $pesteraliasfile)
+		# Note: A 'nicer' test would maybe be to check if the parsed string is in the output of get-alias,
+		# however, the way in which get-alias outputs doesn't have a nice contains() method which can check this (as far as I could see) so I therefore chose to leave this for future work.
```

Comment:
We could check that `get-alias -ErrorAction SilentlyContinue` returns null before import-alias and returns a value after.

Reply:
I like keeping one assertion per test case. Therefore I've added a new test that checks whether `get-alias` should return null when we haven't imported instead of checking both before and after.

Reply:
@iSazonov Can I resolve this?